whois-commons/src/main/java/net/ripe/db/whois/common/DateTimeProvider.java

```diff
@@ -7,14 +7,11 @@
 
 public interface DateTimeProvider {
 
-    /** Return the current date */
-    LocalDate getCurrentDate();
```

Comment:
Do you need to rename the methods? It causes a huge diff in the PR (also in whois-internal)

Reply:
I noticed the huge diff also :/
My preference is to rename, as this makes the timezone return by these methods explicit...something which I think is handy especially given the fact we could be talking about two different timezones...

Maybe a compromise is to rename the class?  `UTCDateTimeProvider`. That should reduce the PR diff. Is that a good middle ground?

Reply:
Good idea, I'd be happy with that! Thanks

Reply:
ok, so renaming the class will also lead to a huge diff. so decided to just update the comments instead mentioning the usage of UTC

Reply:
Wouldn't it be better to have a UTCDateTimeProvider as implementation class rather than renaming the interface?
So we have a DateTimeProvider for which we currently have 2 implementations: ClockDateTimeProvider (which is based on system timezone) and TestDateTimeProvider, and then adding a fixed (UTC) timezone DateTimeProvider impl